src/ceph-crash.in

```diff
@@ -0,0 +1,83 @@
+#!@PYTHON_EXECUTABLE@
+# -*- mode:python -*-
+# vim: ts=4 sw=4 smarttab expandtab
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+import time
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-p', '--path', default='/var/lib/ceph/crash',
+        help='base path to monitor for crash dumps')
+    parser.add_argument(
+        '-d', '--delay', default=10.0 * 60, type=float,
+        help='minutes to delay between scans (0 to exit after one)',
+    )
+    return parser.parse_args()
+
+
+def post_crash(path):
+    pr = subprocess.Popen(
+        args=['ceph', 'crash', 'post', '-i', '-'],
```

Comment:
Since the mgr (or entire cluster) might not be up currently, maybe simply prefixing this with "timeout 30" is enough to make this time out and retry gracefully?